params/blue_truck_params.yaml

```diff
@@ -247,7 +247,7 @@ vesc_to_odom_node:
   ros__parameters:
     odom_frame: odom
     base_frame: base_link
-    speed_to_erpm_gain: 4100.0
+    speed_to_erpm_gain: 7317.8
```

Comment:
do we even want to merge this if we plan on going back to 2nd gear to avoid the gearbox lag issues?